build.rs

```diff
@@ -284,7 +285,7 @@ fn main() {
         .file("tectonic/XeTeX_ext.c")
         .file("tectonic/xetexini.c")
         .file("tectonic/XeTeX_pic.c")
-        .define("HAVE_TM_GMTOFF", "1")
+        // .define("HAVE_TM_GMTOFF", "1")
```

Comment:
When I was making the conda-forge packages for the recent release I ran into this issue too. It looks like `tm_gmtoff` is a BSD/GNU extension, so we shouldn't rely on it, so I think it is best just to remove this line altogether.